### PR TITLE
tidy-viewer (new formula)

### PR DESCRIPTION
**Pull request by alexhallam**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/tidy-viewer.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  homepage "https://github.com/alexhallam/tv"
+  url "https://github.com/alexhallam/tv"
+  version "0.0.20"
+  sha256 "98ae51622763d8ef62fd163dcec88842dcfc0c1cdcbdaf2c5d8ae72d63858cb5"
+  license "UNLICENSE"
+
+  def install
+    bin.install "tidy-viewer"
+  end
+  test do
```
Comment (iMichka) [inline Formula/tidy-viewer.rb:12]: Add a newline between the install and test bloc

**Comment by iMichka** [comment]
```
  * Formula tidy-viewer contains non-standard SPDX licenses: ["UNLICENSE"].
    For a list of valid licenses check: https://spdx.org/licenses/
```

**Comment by iMichka** [comment]
Can we compile this from source?

**Review by SMillerDev — CHANGES_REQUESTED** [review]
Please see https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

**Pull request closed without merging**